test.js

```diff
@@ -569,3 +569,16 @@ if (Promise.prototype.finally) {
 		t.is(result.message, 'called');
 	});
 }
+
+test('the spawned process is cancelable', t => {
+	const spawned = execa('ls');
```

Comment:
`ls` is not available on Windows. You should use one of the test fixtures e.g. `node fixtures/forever`